docs/docs/user-guide/training-data-format.mdx

```diff
@@ -0,0 +1,659 @@
+---
+id: training-data-format
+sidebar_label: Training Data Format
+title: Training Data Format
+description: Description of the YAML format for training data
+---
+
+## Overview
+
+### YAML Format
```

Comment:
I'd remove this headline and put the text under `overview`. I think we should either add a paragraph or note on old data formats with a link to their documentation

Reply:
added note; not sure where links should point